### PR TITLE
UNG-2579 - release keycloak connection while obtaining Response

### PR DESCRIPTION
**Pull request by wojciechUrbanski**

### CODE REVIEW

File: src/main/scala/com/ubirch/services/keycloak/users/KeycloakUserService.scala [hunk 1]
```diff
@@ -218,7 +218,8 @@ class DefaultKeycloakUserService @Inject() (keycloakConnector: KeycloakConnector
             .getKeycloak(instance)
             .realm(realm.name)
             .users()
-            .delete(user.getId)))
+            .delete(user.getId)
+            .close()))
```
Comment (lalottacontinua) [inline src/main/scala/com/ubirch/services/keycloak/users/KeycloakUserService.scala:222]: you probably checked, if there are any other keycloak methods, that also would need a close, haven't you? :)

**Review by lalottacontinua — APPROVED** [review]
nice finding!

File: src/main/scala/com/ubirch/services/keycloak/users/KeycloakUserService.scala (hunk 1 above)
Comment (wojciechUrbanski) [reply src/main/scala/com/ubirch/services/keycloak/users/KeycloakUserService.scala:222]: I did ;) Besides the ones that use the `processCreationResponse` method, this is the only method that would need to close the response manually

**Review by Hiroki6 — APPROVED** [review]
awesome! thanks!

**Pull request merged**